client/webserver/site/src/html/wallets.tmpl

```diff
@@ -45,7 +45,10 @@
                   <table id="balanceTable" class="fs15 mt-2 w-100">
                     <tbody id="balanceDetailBox">
                       <tr id="balanceDetailRow">
-                        <td data-tmpl="category" class="grey"></td>
+                        <td data-tmpl="category" class="grey">
+                         <span></span>
```

Comment:
Add a `data-tmpl` attribute so we don't have to do the `firstElementChild` thing.